packages/esm-patient-orders-app/src/components/medication-record.component.tsx

```diff
@@ -46,12 +46,10 @@ const MedicationRecord: React.FC<MedicationRecordProps> = ({ medication }) => {
           {medication.dosingInstructions && <span> &mdash; {medication.dosingInstructions.toLocaleLowerCase()}</span>}
         </p>
         <p className={styles.bodyLong01}>
-          {medication.orderReasonNonCoded ? (
-            <span>
-              <span className={styles.label01}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
-              {medication.orderReasonNonCoded}
-            </span>
-          ) : null}
+          <span>
+            <span className={styles.label01}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
+            {medication.orderReasonNonCoded ?? t('none', 'None')}
+          </span>
```

Comment:
I actually think the previous implementation here is better. Can we reverse this please?

Reply:
I see. I think I'd prefer just tracking whether or not we need to render the em-dash.

Reply:
The rule I'm going for is consistency: we should either render a "placeholder" for every field without data or, if, as we've done, we only selectively render fields when the convey some piece of information, we should do so consistently here.